tests/foreman/api/test_rhci.py

```diff
@@ -0,0 +1,125 @@
+"""Unit tests for the ``fusor/api/v21/deployments`` paths."""
+from ddt import data, ddt
+from fauxfactory import gen_string
+from nailgun import entities
+from requests.exceptions import HTTPError
+from robottelo.test import APITestCase
+# (too-many-public-methods) pylint:disable=too-many-public-methods
+
+
+@ddt
+class RHCIDeploymentTestCase(APITestCase):
+    """Tests for the RHCI deployment API."""
+
+    @classmethod
+    def setUpClass(cls):
+        """ Setup the base entities for all tests. """
+        super(RHCIDeploymentTestCase, cls).setUpClass()
+        cls.org = entities.Organization().create_json()
+        cls.lc_env = entities.LifecycleEnvironment(
+            organization=cls.org['organization']['id']).create()
+        cls.rhev_engine_host = entities.Host(id=3)
+        # cls.rhev_engine_host = entities.Host(
+        #     organization=cls.org['organization']['id']).create()
```

Comment:
waiting on dev response for me to completely remove this. will add a comment to the bug, if needed.